### PR TITLE
build(nix): ghc922 -> ghc923

### PR DESCRIPTION
**Pull request by teto**
bumped nixpkgs and add support for ghc923 (replacing ghc922)

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/3049"></a>

### CODE REVIEW

**Review by michaelpj — APPROVED** [review]

**Comment by teto** [comment]
this CI reminds me of the kraken. Is it possible to resume the "Nix / build (pull_request)" please ?

**Comment by michaelpj** [comment]
I think it's fine, it's just waiting for some long checks after the merge. Mergify should handle it.

**Comment by teto** [comment]
yeah I was a bit surprised you merged it ^^'' but in a way I was relieved because the CI is slow. I suppose they run on the free tier, any chance we could use the runners from the GHC repo ? The cabal builds could use some caching too, I dont know if the nix builds are cached or not ? (maybe via cachix)

**Pull request merged**
